### DISCUSSION TITLE
Quick Swipe Through Notifications

### DISCUSSION THREAD

**Discussion by brenmcnamara** [community/community#13350] [Mobile]
Love the app! Some quick feedback, on the notifications tab, we can swipe left / right to save / complete the notification. Would go much faster if the action happened on the swipe. So swipe to mark done, instead of swipe to show the done button then tap the button to mark as done

**Comment by aconsuegra** [comment]
Thanks for the feedback @brenmcnamara! Just to confirm, are you using the iOS or Android app?
